packages/manager/apps/container/src/navbar/account.jsx

```diff
@@ -11,6 +11,7 @@ function NavbarAccount({ user }) {
   const { t } = useTranslation(TRANSLATE_NAMESPACE);
   const shell = useShell();
   const uxPlugin = shell.getPlugin('ux');
+  const trackingPlugin = shell.getPlugin('tracking');
```

Comment:
Any reason sometime we directly call `shell.plugin('tracking')` sometimes we declare it as a variable ? Might be worth being consistent